crates/wash/src/bin/wash.rs

```diff
@@ -480,6 +485,14 @@ async fn main() {
         }
     };
 
+    // print info on new wash version if available
+    if cli.experimental {
+        if let Err(e) = inform_new_wash_version().await {
+            eprintln!("Error retreiving new wash version: {}", e);
```

Comment:
Mhhh just realised that this error message might be misleading as to what was being done when it failed. Maybe something like "Error while checking for new wash version:" might be better...